packages/react-player/src/react-player.tsx

```diff
@@ -182,7 +183,10 @@ export const DotLottiePlayer: React.FC<DotLottiePlayerProps> = ({
 
   useUpdateEffect(() => {
     if (typeof src !== 'undefined') {
-      dotLottiePlayer.updateSrc(src);
+      const dl = new DotLottiePlayerCommon(src, container.current, config);
+
+      dl.load();
+      setDotLottiePlayer(dl);
```

Comment:
@afsalz, isn't this dotLottiePlayer initialization code already abstracted in the useDotLottiePlayer? Can't we just export it for reuse from the hook instead of exposing `setDotLottiePlayer`?

```js
...
const initDotLottiePlayer = useCallback(() => {
    const dl = new DotLottiePlayer(src, container.current, config);
    dl.load();
    return dl;
  }, [container]);
  
  return {
    dotLottiePlayer,
    initDotLottiePlayer
  }
```